official/resnet/keras/README.md

```diff
@@ -35,23 +43,43 @@ provide a few options.
 
 Once your dataset is ready, you can begin training the model as follows:
 
+```bash
+python keras_imagenet_main.py 
+```
+
+Again, if you did not download the data to the default directory, specify the
+location with the `--data_dir` flag:
+
 ```bash
 python keras_imagenet_main.py --data_dir=/path/to/imagenet
 ```
 
-There are more flag options you can specify. See 
-[`keras_common.py`](keras_common.py) for full list of options.
+There are more flag options you can specify. Here are some examples:
+
+- `--use_synthetic_data`: when set to true, synthetic data, rather than real 
+data, are used;
+- `--batch_size`: the batch size used for the model;
+- `--model_dir`: the directory to save the model checkpoint;
+- `--train_epochs`: number of epoches to run for trianing the model;
+- `--train_steps`: number of steps to run for training the model. We now only
+support a number that is smalle than the number of bathces in an epoch.
+- `--skip_eval`: when set to true, evaluation of the model will be skipped. 
+
```

Comment:
It might be nice to provide a full command line that we often use. Maybe something like this:

For example, this is a typical command line to run with ImageNet data with batch size 128 per GPU:

Reply:
Done.